src/Raven.Studio/typescript/viewmodels/shell/about.ts

```diff
@@ -183,11 +183,25 @@ class about extends viewModelBase {
            return "icon-cancel";
         });
     }
+    
+    renewLicenseConfigurationSettings = ko.observable<boolean>(false);
```

Comment:
those names are misleading. It isn't settings, but flag if user can renew license, right?
I'd suggest name: canRenewLicense, canActivateLicense, etc.

Reply:
Well - no - it is not representing a can/can't flag 
The flags that are responsible for that are in:
https://github.com/ravendb/ravendb/blob/726ce180fc0a7f102d63ddab55f048dfb5e942f5/src/Raven.Studio/typescript/common/shell/accessManager.ts#L32

These new params represent the content of the settings.json file, 
or whatever is set with the env variables, or command line args
i.e.  "License.CanRenewLicense": false

Reply:
renamed to:
```
isActivateLicenseEnabled
isRenewLicenseEnabled
isForceUpdateEnabled
```